scripts/library.sh

```diff
@@ -418,6 +418,12 @@ function is_protected_gcr() {
   [[ -n $1 && "$1" =~ "^gcr.io/knative-(releases|nightly)/?$" ]]
 }
 
+# Return whether the given parameter is the knative-prow cluster.
+# Parameters: $1 - Kubernetes cluster context (output of kubectl config current-context)
+function is_protected_cluster() {
+  [[ -n $1 && "$1" = "gke_knative-tests_us-central1-f_prow" ]]
```

Comment:
Will it be better to define "gke_knative-tests_us-central1-f_prow" as a var [here](https://github.com/knative/test-infra/blob/master/scripts/library.sh#L20)?

Reply:
prefer not to put it up there, as:
1. this variable is not used anywhere else, and it's not meant to be
1. consistent with is_protected_gcr above